ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -57,29 +67,29 @@ define([
                     this.get("disabledConfigurations").remove(configuration);
                 } else if (configuration === this.get("currentConfiguration")) {
                     this.stopListening(configuration);
+                    this.stopPolling();
                     this.set("currentConfiguration", undefined);
                 }
             },
             setCurrentConfiguration: function (configuration) {
                 var sync = 'sync';
                 var statusUpdate = 'status:update-';
-                this.set({
```

Comment:
Were we just setting the configuration twice? 😄

Reply:
Yep 🤦‍♂️